plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/DeltaLakeSchemaSupport.java

```diff
@@ -105,13 +109,17 @@ private DeltaLakeSchemaSupport() {}
     private static final String INVARIANTS_FEATURE_NAME = "invariants";
     public static final String TIMESTAMP_NTZ_FEATURE_NAME = "timestampNtz";
     public static final String VACUUM_PROTOCOL_CHECK_FEATURE_NAME = "vacuumProtocolCheck";
+    public static final String VARIANT_TYPE_FEATURE_NAME = "variantType";
+    public static final String VARIANT_TYPE_PREVIEW_FEATURE_NAME = "variantType-preview";
```

Comment:
This feature is still in closed source in Delta right?
I don't see it in https://github.com/delta-io/delta/blob/master/PROTOCOL.md#valid-feature-names-in-table-features

Reply:
Right. We should add a product test once we get the environment. https://github.com/trinodb/trino/issues/22342 is the follow-up issue.